src/schema/v2/__tests__/user.test.ts

```diff
@@ -356,6 +356,143 @@ describe("User", () => {
     })
   })
 
+  describe("purchasesConnection", () => {
+    it("returns user purchases", async () => {
+      const query = `
```

Comment:
I ended up leaving `image` out of this _spec query_ as I was not able to stub out the structure for building out the correct context.